.circleci/config.yml

```diff
@@ -472,19 +440,15 @@ commands:
                 command: circleci-agent step halt
 
 jobs:
-  build-and-deploy-job:
+  app-deploy:
```

Comment:
Making job name simpler and follow job naming convention: Name begins with a noun.